test/sagemaker_tests/huggingface/inference/requirements.txt

```diff
@@ -27,4 +27,4 @@ fabric
 invoke
 gitpython
 toml
-transformers==4.28.1
+transformers==4.46.0
```

Comment:
Should this be 4.46.1? I see an image in testing repo with `669063966089.dkr.ecr.us-west-2.amazonaws.com/pr-huggingface-pytorch-inference:2.3.0-transformers4.46.0-gpu-py311-cu121-ubuntu20.04-pr-4392`, could be the transformer version unmatch is the reason of not finding the image.

Reply:
Good catch @Captainia, let me fix it!